app/Http/Controllers/StatusController.php

```diff
@@ -150,9 +151,9 @@ public static function getGlobalDashboard(): Paginator {
                                ->orWhere('users.id', $user)
                                ->orWhere(function($query) {
                                    $followings = Auth::check() ? auth()->user()->follows()->select('follow_id') : [];
-                                   $query->where('visibility', StatusVisibility::FOLLOWERS)
-                                         ->whereIn('users.id', $followings)
-                                         ->orWhere('visibility', StatusVisibility::PUBLIC);
+                                   $query->whereIn('users.id', $followings)
+                                         ->where('visibility', StatusVisibility::PUBLIC)
+                                         ->where('visibility', StatusVisibility::FOLLOWERS);
```

Comment:
This cannot work?

Reply:
Why not?

Reply:
If visibility is public AND followers?

Reply:
Ohhh. So maybe 
```php
->whereNot('visibility', StatusVisibility::Private)
->whereNot('visibility', StatusVisibility::Unlisted)
```

Or  a 
```php
->where(function ($query) {
    $query->where('visibility', StatusVisibility::Public)
               ->orWhere('visibility', SatusVisibility::Followers)
});
```
?
What would be the better solution?